src/main/java/net/openhft/chronicle/core/Jvm.java

```diff
@@ -176,7 +176,8 @@ public final class Jvm {
             logger.warn("Resource tracing is turned on. If you are performance testing or running in PROD you probably don't want this");
         REPORT_UNOPTIMISED = Jvm.getBoolean("report.unoptimised");
 
-        PerformanceTuning.reportIssues();
+        if (!Jvm.getBoolean("disable.performance.tuning"))
```

Comment:
I'd name property `disable.performance.tuning.report`
```suggestion
        if (!Jvm.getBoolean("disable.performance.tuning.report"))
```